src/models/addw4/internal-overview.md

```diff
@@ -0,0 +1,32 @@
+# Adder WS (Internal Overview)
+
+## Bottom components and connectors:
+
+![Internal Components and Connectors](./img/components-highlighted.webp)
+
+- LCD panel connector (under the heatsink) is highlighted in light green
+- Keyboard adhesive access point is highlighted in pink
+- RAM slots are highlighted in navy
+- Fan connectors are highlighted in red
+    - Left fan connector (on the right) is below SSD slot #3
+- Speaker connectors are highlighted in yellow
+    - Right fan connector (on the left) is below SSD slot #2
+- Webcam/microphone connector is highlighted in cyan
+- Wireless card is highlighted in purple
```

Comment:
Purple used for wifi card looks very similar to the pink used on keyboard adhesive access point. Could you make the purple more purple?